src/components/menu-panel/index.js

```diff
@@ -4,8 +4,8 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Icon from '../icon';
 
-class SidebarItem extends Component {
-  static displayName = 'SidebarItem'
+class MenuPanelItem extends Component {
```

Comment:
I'm sorry but I don't remember. Why did you change this to MenuPanel? What was wrong with Sidebar?

Reply:
Just not to conflict with CE sidebar... I don't remember either clearly what was the issue exactly.

Reply:
I may reset this commit. I will doublecheck if it really necessary to use different name or not.

Reply:
I found this TODO, this may give an explanation:
https://github.com/purposeindustries/content-editor/blob/71b676fab59825c2658227d4b2eec68faa96f3e9/src/client/js/components/sidebar.js#L1